### PR TITLE
Updated params to hopefully deploy to heruko

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: app/config/production_parameters.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# This file is auto-generated during the composer install
+parameters:
```
Comment (eddiejaoude) [inline app/config/production_parameters.yml]: I would not commit these to the repo, especially as it is **open source**. These should be ENV vars.

*Note: Reset these values*

Comment (brettminnie) [inline app/config/production_parameters.yml]: Thanks for this, am trying to get it working at present, can't get it to
connect to the db instance :/

Comment (eddiejaoude) [inline app/config/production_parameters.yml]: Oh ok. What is the error?

Comment (brettminnie) [inline app/config/production_parameters.yml]: Now we have the compiled artifact is too big, which is strange, on disc
it's ~170 megs, so how that grows to 370 after compilation and compression
is beyond me

Have a hunch though

On Fri, Nov 13, 2015 at 3:47 PM, Eddie Abou-Jaoude <notifications@github.com

**Pull request merged**